msg/FortHealth.msg

```diff
@@ -0,0 +1,12 @@
+uint8 battery_level               # 1-100%, in 10% increments
+bool battery_charging             # 0/1 "bool"
+uint8 connection_strength_vsc     # low 0/1/2 high : as percieved by VSC
+uint8 connection_strength_src     # low 0/1/2 high : as percieved by SRC
+uint8 rssi_vsc                    # documentation says "Reserved for future use"
+uint8 rssi_src                    # documentation says "Reserved for future use"
+uint8 vsc_mode  # 0 = Error
+                # 4 = Local/Searching
+                # 6 = Remote Connected
+                # 9 = Remote Operational
+                # 10 = Menu
+                # 11 = Pause
```

Comment:
Great message, great documentation.

I'm not sure about the name `FortHealth.msg`. FORT is the name of the company, and this is specifically the SRC (product) that we have the health for, so maybe we should call it something like `SrcHealth.msg`?

Reply:
Actually, since the official documentation calls it "RemoteStatus", let's use that terminology and call this `RemoteStatus.msg`

Reply:
@AGummyBear 
I steered away from calling it `RemoteStatus.msg` because I also wanted to include the vsc mode which comes in through the heartbeat message.
We think the VscMode provide some nice debugging info.

VscMode can also be published separately, but then system status will need that extra subscriber.

I was between `SafetyHealth` (not clear enough) or `HriSafetyHealth` since both the SRC and VSC health are in the message. 
`SrcHealth.msg` could work though.